src/ngraph/ngraph_emitter.cc

```diff
@@ -1365,6 +1365,25 @@ void Emitter::CreateLayerOps() {
     return ngraph::builder::make_with_numpy_broadcast<ngraph::op::Add>(
         convolution, bias_reshape);
   };
+  ngraph_op_funcs_["Deconvolution"] =[this](const NodePtr& node) -> NgraphNodePtr {
+    NgraphNodePtr data = op_map_[node->inputs_[0]];
+    NgraphNodePtr filter = op_map_[node->inputs_[1]];
+
+    // N, channel_in, d1,...,dn
+    const auto data_shape = data->get_shape();
+    // channel_out, channel_in/groups, f1,...,fn
```

Comment:
I really like that your comments explain the role of the axes comprising these shapes.

Would it be possible to use some letter other than `n` in the expressions `dn` and `fn`, and in the variable named `n` below?  I found it slightly confusing, given that upper-case `N` means (I assume) batch size.

Reply:
:) I copied this out of the nGraph implementation, but I can modify.